src/SFA.DAS.EmployerIncentives.Commands/ApprenticeshipIncentive/PaymentProcess/SendMetricsReportEmailCommand.cs

```diff
@@ -0,0 +1,37 @@
+﻿using SFA.DAS.EmployerIncentives.Abstractions.Commands;
+using SFA.DAS.EmployerIncentives.Abstractions.Logging;
+using SFA.DAS.EmployerIncentives.Infrastructure.DistributedLock;
+using System;
+
+namespace SFA.DAS.EmployerIncentives.Commands.ApprenticeshipIncentive.PaymentProcess
+{
+    public class SendMetricsReportEmailCommand : DomainCommand, ILockIdentifier, ILogWriterWithArgs
```

Comment:
Shouldn't need the ILockIdentifier interface because there are no likely concurrency issues I think.

Reply:
Fixed